### PR TITLE
[docs] introduces of Doris Pipeline Connector

### PR DESCRIPTION
**Pull request by JNSimba**
pr #2729 has added DorisDataSink, so here is an introduction to its usage and parameters.

### CODE REVIEW

**Comment by JNSimba** [comment]
Already modified, Thanks @gtk96

File: docs/content/connectors/CDC-Pipeline-Connectors/sink/doris.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+       <td>Boolean</td>
+       <td> Whether to use the batch method to write to Doris </td>
+     </tr>
+     <tr>
+           <td>sink.flush.queue-size</td>
+           <td>optional</td>
+           <td style="word-wrap: break-word;">2</td>
+           <td>Integer</td>
+           <td> Queue size for batch writing
+           </td>
```
Comment (lvyanquan) [inline docs/content/connectors/CDC-Pipeline-Connectors/sink/doris.md:111]: These lines do not match the previous indentation.

**Comment by JNSimba** [comment]
Already modified, Thanks @gtk96 @lvyanquan

**Comment by lvyanquan** [comment]
LGTM. add this two docs to docs/content/pipelines/index.md please.

**Review by lvyanquan — APPROVED** [review]
LGTM.

**Comment by leonardBang** [comment]
@JNSimba Could you help rebase this PR ?

**Comment by JNSimba** [comment]
Already modified, Thanks @leonardBang

**Review by leonardBang — APPROVED** [review]
Thanks @JNSimba for the update, LGTM

**Pull request merged**